crypto/verifier.go

```diff
@@ -57,17 +55,16 @@ func VerifySignedLogRoot(pub crypto.PublicKey, hash crypto.Hash, r *trillian.Sig
 
 }
 
-// VerifyObject verifies the output of Signer.SignObject.
-func VerifyObject(pub crypto.PublicKey, hash crypto.Hash, obj interface{}, sig *sigpb.DigitallySigned) error {
-	j, err := json.Marshal(obj)
-	if err != nil {
-		return err
+// VerifySignedMapRoot verifies the signature on the SignedMapRoot.
```

Comment:
Not clear from this documentation why it is also returning a MapRootV1.

Reply:
added docs